src/main/java/gov/cms/madie/models/access/AclOperation.java

```diff
@@ -0,0 +1,24 @@
+package gov.cms.madie.models.access;
+
+import jakarta.validation.constraints.NotEmpty;
+import jakarta.validation.constraints.NotNull;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@Data
+@Builder(toBuilder = true)
+@NoArgsConstructor
+@AllArgsConstructor
+public class AclOperation {
+  @NotEmpty private List<AclSpecification> acls;
+  @NotNull private AclAction operation;
```

Comment:
might have been an error in the JIRA, but could we rename this variable to "action" rather than "operation" to better align with the type?

Reply:
done